ci-operator/step-registry/upi/install/azure/upi-install-azure-ref.yaml

```diff
@@ -9,8 +9,23 @@ ref:
       memory: 100Mi
   env:
   - name: OCP_ARCH
-    default: "x86_64"
+    default: "amd64"
     documentation: |-
-      The architecture of the control plane nodes (e.g., x86_64, aarch64)
+      The architecture of the control plane nodes (e.g., amd64, arm64)
+  - name: CONTROL_PLANE_INSTANCE_TYPE
+    default: ""
+    documentation: |-
+      The instance size of the control plane nodes (e.g., Standard_D8s_v3)
+  - name: COMPUTE_NODE_TYPE
```

Comment:
The issue is that COMPUTE_NODE_TYPE is already defined in other steps of the involved workflows, while the others aren't . I was unsure to do the contrary but also I'd like that we use the same naming across cloud providers. So, (a) COMPUTE_NODE_TYPE is used in some other steps related to Azure, (b) the others are the ones used by AWS UPI.

I'm ok with any solution here: ideally having more consistency would be great. WDYT?

Reply:
I don't feel strongly about this, it was just a nitpick. I didn't know about the usage in other files. I think it would be good to keep some consistency at least in the Azure steps/files, so maybe `{BOOTSTRAP/CONTROL_PLANE}_NODE_TYPE` is an Ok-ish compromise.